packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -255,6 +255,7 @@ const style = StyleSheet.create({
     padding: 3,
     marginTop: 1, // 6 vs 5 top vs bot space difference
   },
+  // @ts-ignore
```

Comment:
What error is it ignoring?

Reply:
this was a really strange one - had a problem with the type of `position`, which i think is because platform.select loses some type info